lintreview/tools/ansible.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
I'm not too sure. This change was recommended by modernize. 
Can you give me an example?